src/parsers/MetadataParser.ts

```diff
@@ -143,7 +173,12 @@ export default class MetadataParser extends StatefulBufferParser {
 
   private parseReforgedPlayerMetadata(): ReforgedPlayerMetadata[] {
     const result: ReforgedPlayerMetadata[] = [];
-    while (this.readUInt8() === 0x39) {
+    const skinSet: Map<number, number[]> = new Map();
+    while (this.peekUInt8() == 0x38 || this.peekUInt8() == 0x39) {
+      const type = this.readUInt8();
+      if (type == 0x38) {
+        console.log("Message Type 0x38 detected (Patch 2.0.2)!");
```

Comment:
Does this serve any other purpose than debugging? If you have a replay that matches this type, please provide a test case instead and remove the logging.

Reply:
Hmm... that's a good shout - I had the idea to log this as it's not a regular situation, but that depends on what Blizzard does in the next patch. I'll remove it and add a test to cover this case